artifacts/deploy/work.karmada.io_clusterresourcebindings.yaml

```diff
@@ -13,8 +13,10 @@ spec:
     kind: ClusterResourceBinding
     listKind: ClusterResourceBindingList
     plural: clusterresourcebindings
+    shortNames:
+    - crb
     singular: clusterresourcebinding
-  scope: Cluster
+  scope: Namespaced
```

Comment:
Wow, unexpected change, and caused by https://github.com/karmada-io/karmada/pull/376/files#r639349073